packages/suite/src/views/settings/index.tsx

```diff
@@ -343,14 +354,22 @@ const Settings = () => {
                                                         variant="tertiary"
                                                         icon="EXTERNAL_LINK"
                                                         alignIcon="right"
+                                                        isDev={isDev()}
```

Comment:
Button.variant = 'danger' would be more consistent

Reply:
It is true, but `danger` button has different size (height, width and font-size), so I would have to change it
![Screenshot 2021-09-23 at 14 36 50](https://user-images.githubusercontent.com/33235762/134507783-f5164cc4-000a-4c13-aac4-a20308cb1c4a.png)

Reply:
But I changed colors to `danger` button 15bc2220e0833bbaaf678e24ccd1dedf849b0128

Reply:
Resolving